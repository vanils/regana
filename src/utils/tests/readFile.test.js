
const readFile = require('../readFile.js');
const path = require('path');

describe('utils', () => {
  describe('#readFile', () => {

    test('should be able to read existing file', () => {
      expect(readFile(path.resolve(__dirname, './mocks/file1.js'))).toBe('const testValue = 1;');
    });

    test('should fail when file does not exist', () => {
      expect(() => {
        readFile(__dirname, './mocks/fileX.js');
      }).toThrow();
    });
  });
});