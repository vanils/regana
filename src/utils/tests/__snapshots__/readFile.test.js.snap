// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils #readFile should fail when file does not exist 1`] = `"Error: file does not exist '/Users/mattimehtonen/Projects/regana/src/utils/tests'"`;
